integration_tests/test_model_download.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+from requests import HTTPError
+
+from kagglehub import model_download
+
+HANDLE = "keras/bert/keras/bert_base_en/2"
+
+
+class TestModelDownload(unittest.TestCase):
+    def test_model_versioned_succeeds(self):
+        expected_path = "kagglehub/models/keras/bert/keras/bert_base_en/2"
```

Comment:
Let's use `bert_tiny_en_uncased`. It is the smallest variation and will run faster.